docs/Getting Started.md

```diff
@@ -0,0 +1,171 @@
+# Getting Started
+Welcome to Inductiva, where large-scale simulations meet simplicity, through a 
```

Comment:
I would be even more straight to the point, and reduce adjectives as much as possible.
Something like:

Welcome to Inductiva, a Python API for running large-scale simulations on the Cloud.